### PR TITLE
fix(styleguide): Set Figma blocks aspect ratio to 16:9 GM-336

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

* These were too short and looked weird.

### PR Checklist

- [x] Related to designs:
- [x] Related to JIRA ticket: https://codecademy.atlassian.net/browse/GM-336
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@43.10.8-alpha.985a86.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://612d1b9df7498714c37dd1f6--gamut-preview.netlify.app>https://612d1b9df7498714c37dd1f6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/612d1b9df7498714c37dd1f6>Deploy Logs</a></p>

File: packages/styleguide/.storybook/components/Docs/DocsPage.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ export const DocsPage: React.FC = ({ children }) => {
         )}
         {design?.url && (
           <GridBox mb={32}>
-            <Figma height="30%" collapsable url={design?.url} />
+            <Figma height={9 / 16} collapsable url={design?.url} />
```
Comment (songstephen) [inline packages/styleguide/.storybook/components/Docs/DocsPage.tsx:121]: The `9 / 16` doesn't render properly, could we swap this to `<Figma height={"56.25%"} collapsable url={design?.url} />`?

image

**Review by songstephen — CHANGES_REQUESTED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@43.10.8-alpha.0e269c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://612d1e09e07c951669e017a5--gamut-preview.netlify.app>https://612d1e09e07c951669e017a5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/612d1e09e07c951669e017a5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@43.10.8-alpha.49b27c.0<br/></pre>

**Review by songstephen — APPROVED** [review]
Thanks for getting that fixed! Really excited for this change :~)

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://612d3dea60675a2df533f019--gamut-preview.netlify.app>https://612d3dea60675a2df533f019--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/612d3dea60675a2df533f019>Deploy Logs</a></p>

**Pull request merged**